flink-kubernetes/pom.xml

```diff
@@ -31,7 +31,7 @@ under the License.
 	<packaging>jar</packaging>
 
 	<properties>
-		<kubernetes.client.version>6.9.2</kubernetes.client.version>
+		<kubernetes.client.version>6.13.4</kubernetes.client.version>
```

Comment:
since it also depends on jackson and there were some package rename in jackson between 2.15.x and 2.16.x